doc/modules/ensemble.rst

```diff
@@ -922,7 +922,7 @@ based on permutation of the features.
    Annals of Statistics, 29, 1189-1232.
 
 .. [Friedman2002] Friedman, J.H. (2002). `Stochastic gradient boosting.
-   <https://statweb.stanford.edu/~jhf/ftp/stobst.pdf>`_.
+   <https://doi.org/10.1016/S0167-9473(01)00065-2>`_.
```

Comment:
This link leads to a page with a restricted access on the paper. I found this link https://citeseerx.ist.psu.edu/document?repid=rep1&type=pdf&doi=48caac2f65bce47f6d27400ae4f60d8395cec2f3, seems unrestricted.